src/rule_definition_tools.jl

```diff
@@ -185,8 +185,10 @@ function scalar_rrule_expr(f, call, setup_stmts, inputs, partials)
         propagation_expr(Δs, ∂s)
     end
 
+    # Multiply output functions have pullbacks with a tuple input, that will be destructured
+    pullback_input = n_outputs== 1 ? first(Δs) : Expr(:tuple, Δs...)
```

Comment:
```suggestion
    pullback_input = n_outputs == 1 ? only(Δs) : Expr(:tuple, Δs...)
```

Reply:
We support Julia 1.0 still

```suggestion
    pullback_input = n_outputs == 1 ? first(Δs) : Expr(:tuple, Δs...)
```